migrations/2_create_assets_up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE assets(
+        PRIMARY KEY(asset_id),
+        FOREIGN KEY (category),
+        status_asset varchar(100) NOT NULL,
+        purchase_date Date NOT NULL,
+        inital_cost varchar(100) NOT NULL,
+        asset_name varchar(100) NOT NULL,
+        specification NOT NULL
```

Comment:
seems like no data type is defined for 'specification'

Reply:
sorry this was json i forgot to save